observability/micrometer/src/main/java/org/wildfly/extension/micrometer/registry/WildFlyRegistry.java

```diff
@@ -34,9 +34,7 @@ default Meter.Id addMeter(WildFlyMetric metric, MetricMetadata metadata) {
         }
     }
 
-    default void close() {
-
-    }
+    void close();
```

Comment:
Not critical, but IMO this should probably just extend `AutoCloseable`. Keeping the method here to remove the exception seems fine. Again, this is not critical.

Reply:
Good call. Done. Updated the switch statement above to newer language standard.